extending.md

```diff
@@ -0,0 +1,44 @@
+# Extending Thunder
+
+### Find an extension
+
+You can find extensions on [Drupal.org](https://www.drupal.org). On [the following page](https://www.drupal.org/project/project_module?f%5B3%5D=drupal_core%3A7234), you can search for `Modules` and on [this page](https://www.drupal.org/project/project_theme?f%5B2%5D=drupal_core%3A7234) for `Themes`. You can find extensive documentation on [Drupal 8 User Guide - Extending and Customizing Your Site](https://www.drupal.org/docs/user_guide/en/extend-chapter.html).
```

Comment:
You can find further information on extensions in the [Drupal 8...

Reply:
👍